package.json

```diff
@@ -61,7 +61,8 @@
     "stylelint": "13.13.1",
     "stylelint-config-recommended-scss": "4.2.0",
     "stylelint-scss": "3.19.0",
-    "webpack-bundle-analyzer": "4.4.2"
+    "webpack-bundle-analyzer": "4.4.2",
+    "typescript": "^4.4.4"
```

Comment:
Hmm... can you put this alphabetically? It seems also that when I run `npm install` on your PR I get `4.4.3` 😕

Reply:
I see it 4.4.4, just checked in the console we have a version for typescript locked to `3.8.3`. will do that now